Scenes/_0main.tscn

```diff
@@ -0,0 +1,13 @@
+[gd_scene load_steps=2 format=2]
```

Comment:
I created this while testing and forgot to delete. Feel free to delete this file.